snow/engine/snowman/transitive.go

```diff
@@ -444,7 +444,7 @@ func (t *Transitive) Start(ctx context.Context, startReqID uint32) error {
 		default:
 			for _, blk := range options {
 				// note that deliver will set the VM's preference
-				if err := t.deliver(ctx, blk, false); err != nil {
+				if err := t.deliver(ctx, t.Ctx.NodeID, blk, false); err != nil {
```

Comment:
Similarly assume this is only after `BuildBlock`?

Reply:
This is called during the startup of the engine. This piece of code is really only for completeness w.r.t. oracle blocks. Specifically, if the `ProposalBlock` is accepted during bootstrapping but neither of the `Option` blocks are - then we initially add the options with our preferences. For large networks like mainnet / fuji this code path probably isn't needed because other nodes would gossip the `Option` blocks and we can just adopt either preference... But this avoids any weird livelock or preference biasing cases.

As far as this goes though - the only options would either be considering ourselves as the block builder or trying to somehow track where we got this block from during bootstrapping (which could cross the restart boundary)